pkg/harvester-manager/l10n/en-us.yaml

```diff
@@ -7,6 +7,19 @@ harvesterManager:
     description: Harvester is a modern Hyperconverged infrastructure (HCI) solution built for bare metal servers using enterprise-grade open source technologies including Kubernetes, Kubevirt and Longhorn.
   plugins:
     loadError: Error loading harvester plugin
+  extension:
+    install:
+      warning: "Harvester extension is not installed"
+      steps:
+        repo:
+          1: Go to
+          2: Manage Repositories
```

Comment:
Change the wording to `App Repositires` or `Repositires` more accurate？

<img width="1475" alt="Screenshot 2024-10-31 at 12 56 25 PM" src="https://github.com/user-attachments/assets/f3813b28-08f5-4267-ac06-748f0c944b65">